Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+[!!LIST (Authorizations)](../UserInterface/01a_ListAuthorizations.md)
+
+
+# Monitor authorizations
+
+An authorization transaction is the permission that your company is allowed to capture money from the customer via a payment service provider. The processing of the authorization transactions works as follows: 
+-   The payment service provider creates an authorization transaction, when a customer presses the *Pay now* button during the order process.   
+- The payment provider sends then the authorization transaction to the *ActindoCore1 Platform*.    
+- In addition, the shop system creates an ID for this transaction.  
+- Actindo links both.   
+
+For Actindo, the authorization transaction serves as a reference to a payment made by a customer.   
+You can use this procedure to display all incoming authorization transactions of a payment service provider. You can check for communication errors between Actindo and the payment service provider that has been caused in a termination of a workflow, for example. In addition, if applicable, you can change the status of a transaction to start continuing further processing or to end it.    
+
+If everything works fine with the posting of a payment, you do not need to monitor the *Authorizations* menu entry. In case of errors you can capture, cancel, or void an authorization transaction manually.   
+> [Info] You can only change the status of an authorization transaction, but not any data.
+
+
+## Check authorization transaction
+An authorization transaction provides numerous payment-related details, which may vary depending on the payment service provider settings. To decide, whether you can manually capture, cancel, or void an authorization transaction, you need to check its details.
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one authorization transaction has been created.
+- It is recommended that you have added the *Status ID* column to your view so that you are able to sort or filter for specific statuses. 
```

Comment:
```suggestion
- It is recommended that you have added the *Status ID* column to your view so that you are able to sort or filter for specific statuses. 
```
Das ist kein Prerequisite. Das kann man, z.B., unter dem entsprechenden Schritt als Info angeben.